plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/DefaultThriftMetastoreClientFactory.java

```diff
@@ -233,4 +295,64 @@ private static void validateCertificates(KeyStore keyStore)
             }
         }
     }
+
+    public static class ThriftHttpContext
```

Comment:
This could be a record

Reply:
I am not sure I understand this. Can you please provide more details?

Reply:
Nevermind, I understood what you meant later.